### PR TITLE
[Core] Initialize cmf_routing by default

### LINKED ISSUE #3963: [Bug] Error "This value is not valid" when tried to create a new page route
I cloned and installed Sylius on my machine.

1. I created a new page.

2. When I tried to create a route for the specified page I got the following error:

```
Message:
This value is not valid.

Origin:
parent

Cause:

Symfony\Component\Validator\ConstraintViolation
Object(Symfony\Component\Form\Form).children[parent] = /cms/routes

Caused by:

Symfony\Component\Form\Exception\TransformationFailedException
Unable to reverse value for property path "parent": Could not transform path "/cms/routes" to document. Path not found.

Caused by:

Symfony\Component\Form\Exception\TransformationFailedException
Could not transform path "/cms/routes" to document. Path not found.
```

### PR DESCRIPTION
**Pull request by okwinza**
Fixes #3963, Sylius/Sylius-Standard#78
Based on @pamil's symfony-cmf/RoutingBundle#318

Should be ok to merge once stable version of RoutingBundle is tagged.

### CODE REVIEW

**Comment by pamil** [comment]
:+1:

**Comment by michalmarcinkowski** [comment]
Let's require [1.4.0-RC1](https://github.com/symfony-cmf/RoutingBundle/releases/tag/1.4.0-RC1) for now, can you update composer?

**Comment by okwinza** [comment]
@michalmarcinkowski done.
Let me know if there is any better version constraint for this case. I'm not quite familiar with composer magic yet :)

File: composer.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
         "symfony-cmf/create-bundle": "~1.2",
         "symfony-cmf/media-bundle": "~1.2",
         "symfony-cmf/menu-bundle": "~1.2",
-        "symfony-cmf/routing-bundle": "~1.2",
+        "symfony-cmf/routing-bundle": "~1.4.0-RC1",
```
Comment (pamil) [inline composer.json]: Let's use `^` instead of `^` so it will allow for future releases `>=1.5` too.

**Comment by okwinza** [comment]
Should be good to go now @pamil @michalmarcinkowski

**Comment by pjedrzejewski** [comment]
@okwinza Build is failing, could you have a look?

**Comment by pamil** [comment]
Lock needs to be updated :)

**Comment by okwinza** [comment]
Yep, totally forgot about lock, thanks for hint too, @pamil!
Fixed, rebased, squashed -- i'd be damned if it's not green this time :D

**Comment by okwinza** [comment]
@pjedrzejewski green :tada:

**Comment by pjedrzejewski** [comment]
Thanks Oleg! :+1:

**Pull request merged**